versions/1/spec.yml

```diff
@@ -19,6 +19,11 @@ spec:
     name: "changelog.yml"
     required: true
     $ref: "./changelog.spec.yml"
+  - description: The package's NOTICE file
+    type: file
+    contentMediaType: "text/plain"
```

Comment:
Is it necessary to define the content type here provided that no action is taken based on this? Did you try without this definition?

Reply:
I just made an assumption that this was necessary. I'm not intimately familiar with the details but am glad to remove if that is preferred.

Reply:
I'm okay leaving this in. We may not take any action on this content type today but if we do in the future, the NOTICE file would automatically get covered.